### ISSUE TITLE
how to config multi disks

### ISSUE THREAD

**Issue by 0ZhangJc0** [apache/pinot#9036]
Support to add multiple directory location for this property in server (pinot.server.instance.dataDir)
Can we add multiple directory locations path to this below server conf property?

pinot.server.instance.dataDir=
Like disk1 and disk2

So server chooses any one during runtime ...

Our server have multi disks, If Pinot does not support configuring multiple data dirss, server resources will be wasted

**Comment by 0ZhangJc0** [comment]
Support to add multiple directory location for this property in server (pinot.server.instance.dataDir)
Can we add multiple directory locations path to this below server conf property?

pinot.server.instance.dataDir=
Like disk1 and disk2

So server chooses any one during runtime ...

Our server have multi disks, If Pinot does not support configuring multiple data dirss, server resources will be wasted

**Comment by kishoreg** [comment]
We were thinking of adding something like this to support multiple tiers as well.. 

Cc @npawar @klsince

**Comment by klsince** [comment]
Yeah, I opened this issue early on and WIP on it: https://github.com/apache/pinot/issues/8843 feel free to leave comment there or tag me here for further discussion.
